src/styles/blocks/card.scss

```
@@ -0,0 +1,80 @@
+@use '../utils/variables' as *;
+
+.card {
+  box-sizing: border-box;
+  padding: 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 16px 3px;
+  }
+
+  &__name {
+    font-size: 1.2em;
+    margin-top: 24px;
+    line-height: 18px;
+    font-weight: $secondary-weight;
+  }
+
+  &__code {
+    font-size: 1em;
+    color: $secondary-color;
+    margin-top: 4px;
+    line-height: 14px;
+  }
+
+  &__stars,
+  &__cost {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &__cost {
+    margin-top: 24px;
+  }
+
+  &__price {
+    font-size: 1.2em;
+    color: $secondary-color;
+    line-height: $line;
+    font-weight: $main-weight;
+    font-weight: 400;
+  }
+
+  &__total {
+    font-size: 1.6em;
+    font-weight: $main-weight;
+    line-height: $line;
+  }
+
+  &__button {
+    height: 40px;
+    width: 166px;
```

use 100% so your code would be more flexible and responsive ;)
```suggestion
    width: 100%;
```